theme/static/js/resourceLandingAjax.js

```diff
@@ -868,10 +869,11 @@ function get_irods_folder_struct_ajax_submit(res_id, store_path) {
             $("#flag-uploading").remove();
             $("#fb-files-container, #fb-files-container").css("cursor", "default");
 
-            if (mode === "edit" && result.hasOwnProperty('spatial_coverage')){
-                var spatialCoverage = result.spatial_coverage;
-                updateResourceSpatialCoverage(spatialCoverage);
-            }
+            // TODO: do we need to upadte SpatialCoverage every time we get irods struct?
```

Comment:
@pkdash at some point can you help me to understand why we refresh coverage values in the UI every time we touch the iRods file structure?

Reply:
Here's where we introduced this behavior:
https://github.com/hydroshare/hydroshare/commit/e616829b5d6000abf5f3fff9332caed6803de3c3

Reply:
@devincowan It may at some point was necessary. Some file uploads with metadata extraction injects coverage metadata at the resource level. Since we do a file structure refresh on file upload, this was used to update the resource level coverage in the UI. I tried to comment out this if block (https://github.com/hydroshare/hydroshare/blob/develop/hs_core/views/resource_folder_hierarchy.py#L222), I did not see any adverse effect on the UI as it relates to display of the resource level coverage. I think we don't need it anymore as the UI is making separate call to get resource level metadata on file upload.